### ISSUE TITLE
Model 1 create

### ISSUE THREAD

**Issue by RaymondBrien** [RaymondBrien/cherry-ml#7]
Model creation using option 2 to start (after refining and confirming correct starting params)

model creation 
from tensorflow.keras.models import Sequential
from tensorflow.keras.layers import Activation, Dropout, Flatten, Dense, Conv2D, MaxPooling2D

OPTION 1 starter:

def create_tf_model():
    model = Sequential()

    model.add(Conv2D(filters=32, kernel_size=(3, 3),  # find only dominant pixels 
              input_shape=image_shape, activation='relu', ))   # uses average image shape from the data visualtion stage 
    model.add(MaxPooling2D(pool_size=(2, 2)))  # reduce size of image by extracting only those dominant features

    model.add(Conv2D(filters=64, kernel_size=(3, 3),
              input_shape=image_shape, activation='relu', ))
    model.add(MaxPooling2D(pool_size=(2, 2)))

    model.add(Conv2D(filters=64, kernel_size=(3, 3),
              input_shape=image_shape, activation='relu', ))
    model.add(MaxPooling2D(pool_size=(2, 2)))

    model.add(Flatten())  # single list of all values, to feed into dense layer
    model.add(Dense(128, activation='relu'))

    model.add(Dropout(0.5)) # drop 50% to avoid overfitting 
    model.add(Dense(1, activation='sigmoid'))  # output layer (only 1 neuron) with probability result

    model.compile(loss='binary_crossentropy',
                  optimizer='adam',
                  metrics=['accuracy'])

    return model

___
OPTION 2 starter:

import tensorflow as tf
from tensorflow.keras import layers, models

def create_powdery_mildew_classifier(input_shape=(224, 224, 3)):
    """
    Creates a CNN model for classifying cherry leaves as healthy or infected with powdery mildew.
    
    Args:
        input_shape: Tuple of (height, width, channels) for input images
        
    Returns:
        tf.keras.Model: Compiled CNN model
    """
    model = models.Sequential([
        # Input Layer and Initial Convolution Block
        layers.Conv2D(32, (3, 3), activation='relu', input_shape=input_shape),
        layers.BatchNormalization(),
        layers.MaxPooling2D((2, 2)),
        
        # Second Convolution Block
        layers.Conv2D(64, (3, 3), activation='relu'),
        layers.BatchNormalization(),
        layers.MaxPooling2D((2, 2)),
        
        # Third Convolution Block
        layers.Conv2D(128, (3, 3), activation='relu'),
        layers.BatchNormalization(),
        layers.MaxPooling2D((2, 2)),
        
        # Fourth Convolution Block
        layers.Conv2D(256, (3, 3), activation='relu'),
        layers.BatchNormalization(),
        layers.MaxPooling2D((2, 2)),
        
        # Flatten and Dense Layers
        layers.Flatten(),
        layers.Dense(512, activation='relu'),
        layers.Dropout(0.5),
        layers.Dense(256, activation='relu'),
        layers.Dropout(0.3),
        layers.Dense(1, activation='sigmoid')  # Binary classification
    ])
    
    # Compile model
    model.compile(
        optimizer='adam',
        loss='binary_crossentropy',
        metrics=['accuracy', tf.keras.metrics.Precision(), tf.keras.metrics.Recall()]
    )
    
    return model

# Example usage:
# model = create_powdery_mildew_classifier()
# model.summary()
